unit_tests/test_socket_connection.py

```diff
@@ -425,7 +425,7 @@ def test_udp_broadcast_client(self):
         self.assertEqual(data_to_send, server.received)
         self.assertEqual(received, server.data_to_send)
 
-    @pytest.mark.skipif(sys.platform == "win32", reason="Raw sockets not supported on Windows.")
+    @pytest.mark.skipif(sys.platform in ["win32", "darwin"], reason="Raw sockets not supported on Windows.")
```

Comment:
should be `not supported on Windows or Mac OS`.

Or alternatively, two separate `skipif`s with their own message.